xero-payroll-nz.yaml

```diff
@@ -4865,7 +4869,8 @@ paths:
                             "effectiveFrom": "2020-05-15T00:00:00",
                             "annualSalary": 101,
                             "status": "Active",
-                            "paymentType": "Salary"
+                            "paymentType": "Salary",
+                            "workPatternType": "DaysAndHours"
```

Comment:
Are these examples automatically generated or manually added?
If manual should we add some examples of `regularWeek` workPatternTypes?

Reply:
Good point, I will change one for `regularWeek`